### PR TITLE
Clarification: the state and not the events are overwritten

### PR DESCRIPTION
**Pull request by AdamBien**
The history of events remains / the event is appended and only the state is overwritten

Preview: https://pr1689--matrix-spec-previews.netlify.app

Fixes: #1686

### CODE REVIEW

File: data/api/client-server/room_state.yaml [hunk 1]
```diff
@@ -20,8 +20,8 @@ paths:
     put:
       summary: Send a state event to the given room.
       description: |
-        State events can be sent using this endpoint.  These events will be
-        overwritten if `<room id>`, `<event type>` and `<state key>` all
+        State events can be sent using this endpoint.  These events will be appended
```
Comment (richvdh) [inline data/api/client-server/room_state.yaml:23]: appended to what?

**Review by richvdh — CHANGES_REQUESTED** [review]
Thank you for your contribution! Please take a look at the [contributing guide](https://github.com/matrix-org/matrix-spec/blob/main/CONTRIBUTING.rst), particularly with respect to [changelog entries](https://github.com/matrix-org/matrix-spec/blob/main/CONTRIBUTING.rst#adding-to-the-changelog) and [sign-off](https://github.com/matrix-org/matrix-spec/blob/main/CONTRIBUTING.rst#sign-off).

File: data/api/client-server/room_state.yaml (hunk 1 above)
Comment (AdamBien) [reply data/api/client-server/room_state.yaml]: +1 we should not prescribe how the events are processed internally. I changed the wording from appended to "processed"

**Comment by richvdh** [comment]
Unfortunately we can't accept this without sign-off.

**Pull request closed without merging**